apps/client/src/common/helpers/axios-helper.ts

```diff
@@ -28,8 +28,6 @@ export const setupAxios = (axiosInstance: AxiosInstance) => {
   axiosInstance.interceptors.response.use(
```

Comment:
### Critical Points:
❌ The code removes the `console.log({ error });` line without providing an alternative way to handle errors. This could lead to missed debugging opportunities and make it harder to identify issues in the future.
❌ The `switch` statement only handles a 400 status code. It's important to consider other potential error codes and implement appropriate handling for them.